proposals/0315-Add-RPC-Conflict-Management.md

```diff
@@ -114,79 +123,44 @@ OEMs can restore the deleted `UI.Slider` and modify its priority. For example, i
-
-(* 2): According to the current SDL regulations, when a certain app has an HMI level of FULL, the HMI level of other similar apps will not be FULL. Therefore, there is a low possibility of RPC conflict with similar apps. Thus, RPC conflict for similar apps is outside the scope of this proposal.<br>
-(* 3): For the app types with the same priority, the first RPC will be prioritized.
-
-Below shows the Json example for the `AppHMIType` priority table:
-
-```json
-"RpcAppTypePriority":{
-    "Navigation:": 1,
-    "Projection:": 1,
-    "Media": 2,
-    "Other": 3
-}
-```
+OEMs can modify the `App priority` table and adjust the priority of application according to their own specifications. In fact, since EMERGENCY is set independently as the highest priority, the priority is determined by the items excluding EMERGENCY.
+For RPCs with the same priority, the HMI Status table, which is described later, will be used to determine the priority.
 
-OEMs can modify any app type priority. For example, if the Media priority is modified to "1", its app type priority will be the same as Navigation and Projection.
 
-```json
-"RpcAppTypePriority":{
-    "Navigation:": 1,
-    "Projection:": 1,
-    "Media": 1,
-    "Other": 3
-}
-```
+<b>HMI Status Table</b><br>
+HMI Status table describes the priority for each app level. When a conflict between RPCs with the same priority in the RPC priority table and `App priority` table occurs, the RPC with the higher priority is determined according to HMI Status table. Below are the default settings of HMI Status table.
 
-OEMs can delete any app type priority. For example, if you delete the Media priority, its app type priority will be the same as Other.
+<b>Table 4.</b> Default settings of HMI Status table
 
-```json
-"RpcAppTypePriority":{
-    "Navigation:": 1,
-    "Projection:": 1,
-    "Other": 3
-}
-```
+| HMI Level (String) | Priority (INT) | Note (String)    |
+|:-:                 |:-:             |:-:               |
+| FULL               | 1              | Highest priority |
+| LIMITED            | 2              |                  |
+| BACKGROUND         | 3              |                  |
+| NONE               | 4              | Lowest priority  |
 
-OEMs can add another app type if needed. For example, if you add an app type such as Communication and set the priority to "2", its app type priority will be the same as Media.
+If two RPCs with the same priority are sent, the second RPC sent will be prioritized.
 
-```json
-"RpcAppTypePriority":{
-    "Navigation:": 1,
-    "Projection:": 1,
-    "Media": 2,
-    "Communication": 2,
-    "Other": 3
-}
-```
 
 #### RPC Conflict Management Module
 The following explains functions of `InterruptManager`.
 
-<b>1.Loading of configuration file</b><br>
-`InterruptManagerConfig` is loaded during the startup of SDL Core.
+<b>1.Loading of configuration table</b><br>
+`InterruptManagerConfig` is loaded from policy table during the startup of SDL Core.
```

Comment:
Should `InterruptManagerConfig` be `RpcPriority` here?

Reply:
@theresalech -san,
Thank you for your response.
In this matter, I think OK as it is.
Because I described "RPC priority table and App Priority table are specified in InterruptManagerConfig." in this proposal.